src/Styles/HomePage.css

```diff
@@ -0,0 +1,44 @@
+p.title{
+    font-weight: bold;
+    text-decoration-line: underline;
+}
+p.info{
+    font-weight: bold;
+    text-decoration-line: underline;
+}
+p.info_grid{
+    display: grid;
+    grid-template-columns: 2fr 3fr;
+    grid-gap: 1px;
+    grid-template-areas: "attributeName attributeValue";
+}
+p.attributeName {
+    font-size: 14px;
+    font-weight: bold;
+    grid-area: attributeName;
+    text-align: left;
+}
+p.attributeValue {
+    font-size: 14px;
+    grid-area: attributeValue;
+    text-align: right;
+}
+p.redfont {
```

Comment:
ohh good idea...will fix this in the next req

Reply:
Also, for js, it's better to use className instead of class. It doesn't effect the code but it's just better syntax. For html, it's better to use class but for js, it's better to use className